dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -38,7 +38,8 @@ check_not_parsing_features() {
     # These two files contain sanity checks using FEATURES_ so are allowed
     pathspec+=(':!Makefile.include' ':!makefiles/info-global.inc.mk')
 
-    git -C "${RIOTBASE}" grep "${patterns[@]}" -- "${pathspec[@]}"
+    git -C "${RIOTBASE}" grep "${patterns[@]}" -- "${pathspec[@]}" \
+        | sed '1i \\nInvalid use of FEATURES_ variables content:'
```

Comment:
I think that the message in the comment above is more clear ("Modules should not check the content of FEATURES_PROVIDED/_REQUIRED/OPTIONAL")

Reply:
Will update.